modules/javafx.base/src/main/java/com/sun/javafx/collections/VetoableListDecorator.java

```diff
@@ -260,6 +267,7 @@ public boolean addAll(int index, Collection<? extends E> c) {
 
     @Override
     public boolean removeAll(Collection<?> c) {
+        Objects.requireNonNull(c);
```

Comment:
Looks like a good change, if `c` was `null` and the backing list was empty, this would not throw NPE, but now it does.